tests/environment/.env

```diff
@@ -1,9 +1,21 @@
+ALFRESCO_TAG=latest
 SOLR6_TAG=2.0.3
 POSTGRES_TAG=13.3
 ACTIVEMQ_TAG=5.16.1
 AIMS_TAG=1.2
 SYNC_SERVICE_TAG=3.7.0
 ES_TAG=7.10.1
+TRANSFORM_ROUTER_TAG=1.5.2
+TRANSFORM_CORE_AIO_TAG=2.5.7
+SHARED_FILE_STORE_TAG=0.16.1
+ACS_NGINX_TAG=3.0.0
+ELASTICSEARCH_TAG=7.10.1
+KIBANA_TAG=7.10.1
+LIVE_INDEXING_TAG=3.1.1-A1
+LIVE_REINDEXING_TAG=3.1.1-A1
```

Comment:
Is this `LIVE_REINDEXING_TAG` variable actually used anywhere?

Reply:
Good spot. I removed the reindexing d-c as it doesn't seem to be needed and forgot the tag 👍